### ISSUE TITLE
Support multiple schemas for schemaFile (or introduce something like schemaDir)

### ISSUE THREAD

**Issue by x80486** [edeandrea/xjc-generation-gradle-plugin#1]
Would be great if there is a way to support multiple schemas when using `schemaFile` – although that name implies only one, so it's probably better to have something like `schemaDir` (or something similar).

This is what I was trying to achieve:

```groovy
xjcGeneration {
  final def basePackage = "tld.domain.base"

  schemas {
    event {
      javaPackageName = "${basePackage}.event"
      schemaFile = "event/Events.xsd"
    }
    multiple {
      javaPackageName = "${basePackage}.multiple"
      schemaFile = "multiple/" // Does not work, it must be a file; neither using /**/*.xsd
    }
  }
}
```

**Comment by edeandrea** [comment]
So in your use case having multiple schema files - are the objects that are created from those schema files related in some way? Or are you just looking for a way to have multiple un-related schemas be generated using the same java package?

I ask because generally if multiple schema files are related, then there is usually 1 "root" schema file that has references to the others. This is fine and fully supported as is.

**Comment by x80486** [comment]
Sometimes they are related, and sometime not...the real constraint is the project setup: it's already configured like that, so I can't really modify all the references. The goal is to use them within the same Java package, regardless of "their relationship".

**Comment by edeandrea** [comment]
I haven't tested this nor is it ideal, but I believe you could have more than one schema with the same javaPackageName, so long as none of the schemas defined an element with the same name. You'd run into an issue with that anyways if there was more than 1 unrelated schema that defined an element by the same name.

I'll have some cycles next week and can take a look at an improvement around this.

**Comment by edeandrea** [comment]
@x80486 This and #3 has been implemented and published to the plugin portal in version 1.1.

**Comment by x80486** [comment]
Thank you so much...I'll give it a try next week once I get back to work (because I have everything there) and will let you know if there is something else that could to be tweaked/improved.
